src/simplify_algebra.cpp

```diff
@@ -41,6 +41,8 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
+MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_TRACE_PASSES);
```

Comment:
Should it be removed ?

Reply:
It is already defined somewhere else

Reply:
I have it in incase people want to debug the passes and get the debug info of what item is failing and causing the exception to be thrown

Reply:
We already have environment var by that name in `pass_manager`

Reply:
I know we do, but I can't get this to compile without adding that to the top of the file for the divide by zero error case